projects/packages/jetpack-mu-wpcom/src/features/block-patterns/class-block-patterns-from-api.php

```diff
@@ -1,19 +1,19 @@
 <?php
 /**
- * Class Wpcom Block Patterns From Api
+ * Class Block Patterns From Api
  *
  * @package automattic/jetpack-mu-wpcom
  */
 
 /**
  * Require the utils class.
  */
-require_once __DIR__ . '/class-wpcom-block-patterns-utils.php';
+require_once __DIR__ . '/class-block-patterns-utils.php';
 
 /**
- * Class Wpcom_Block_Patterns_From_Api
+ * Class Block_Patterns_From_Api
  */
-class Wpcom_Block_Patterns_From_Api {
+class Block_Patterns_From_Api {
```

Comment:
I noticed that `Block_Patterns_From_Api` and `Block_Patterns_Utils` were renamed. I'm curious about a rationale behind it. 😃

Reply:
I found that other classes in the package are not prefixed. Only functions are prefixed so I followed that convention.

Reply:
Other classes in our codebase seem to utilize namespaces, for example;
https://github.com/Automattic/jetpack/blob/8dc6c3fa551c412a7306a5caec087822c17bf72d/projects/packages/jetpack-mu-wpcom/src/features/coming-soon/coming-soon.php#L8
https://github.com/Automattic/jetpack/blob/8dc6c3fa551c412a7306a5caec087822c17bf72d/projects/packages/jetpack-mu-wpcom/src/features/domain-email-nag/domain-email-nag.php#L8

While some classes (such as Launchpad) are not prefixed, their names are sufficiently unique to our dotcom environment, in my opinion. 
Using a prefix, namespace, or unique name might be beneficial to prevent common accidents in WordPress, just in case. Maybe I'm too worried.🙂🙈 WDYT? @miksansegundo

Reply:
Okay, let's prefix those classes. Please push a commit to contribute to this PR or merge it into yours with that change. 🙏